src/Sylius/Bundle/CoreBundle/OrderProcessing/OrderTaxesApplicator.php

```diff
@@ -0,0 +1,176 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\OrderProcessing;
+
+use Sylius\Bundle\CoreBundle\Distributor\IntegerDistributorInterface;
+use Sylius\Bundle\CoreBundle\Provider\DefaultTaxZoneProviderInterface;
```

Comment:
Maybe this should go to ``Taxation`` namespace but definitely in the component, not core bundle!